.github/workflows/release-4-publish-release.yml

```diff
@@ -0,0 +1,359 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: Release - 4 - Publish Release After Vote Success
+
+on:
+  workflow_dispatch:
+    inputs:
+      dry_run:
+        description: 'Dry run mode (check to enable, uncheck to perform actual operations)'
+        required: false
+        type: boolean
+        default: true
+      staging_repository_id:
+        description: 'Nexus staging repository ID to release (e.g., orgapachepolaris-1234)'
+        required: true
+        type: string
+
+jobs:
```

Comment:
Mind adding a final job (maybe in a follow-up) to print the proposed `[ANNOUNCE]` email subject and content?

Reply:
I cannot find a target to write the announcement?  So I would say definitely in a follow up.

Reply:
I'm ok to do it in a follow-up.

Just throwing in arbitrary, not fleshed-out ideas here...
1. Use a GH issue to track release progress and let the workflow add a comment with the email suggestions there.
    Requires some GH API interaction - either REST (`gh api ...`) or [scripted](https://github.com/actions/github-script).
2. Emit the email suggestions as a GH step summary
3. Both?